src/crypto/tls/handshake_client_tls13.go

```diff
@@ -120,6 +125,40 @@ func (hs *clientHandshakeStateTLS13) handshake() error {
 		}
 	}
 
+	// Check for ECH acceptance confirmation.
+	if c.ech.offered {
+		echAcceptConfTranscript := cloneHash(hs.transcriptInner, hs.suite.hash)
+		if echAcceptConfTranscript == nil {
+			c.sendAlert(alertInternalError)
+			return errors.New("tls: internal error: failed to clone hash")
+		}
+
+		sh := hs.serverHello.marshal()
+		echAcceptConfTranscript.Write(sh[:30])
+		echAcceptConfTranscript.Write([]byte{0, 0, 0, 0, 0, 0, 0, 0})
```

Comment:
I think you can set the length and assing `0` just once...

Reply:
how does this work, syntactically?

Reply:
Fixed this by declaring a global `var zeros = [8]byte{}` and referring to it here and elsewhere.